### ISSUE TITLE
how to train mask-r-cnn

### ISSUE THREAD

**Issue by JackSparrow3** [fabienbaradel/object_level_visual_reasoning#5]
Hi Fabien,
thank you for your works!
the paper doesn't mention how to train a mask-r-cnn since VLOG dataset does has any mask annotation, i wonder if your team has annotate the mask by yourself or use other open datasets to train mask-r-cnn? 
thanks again!

**Comment by fabienbaradel** [comment]
Hi @JackSparrow3 
Thanks :)
You can find the annotations here: https://fabienbaradel.github.io/masks_data/
We have used MaskRCNN pretrained on COCO. We did not train MaskRCNN on our datasets (VLOG, Something Something, EPIC) because we do not have access to object labels. We have only used the prediction made by Maskrcnn.
I hope it is clear now.
In case you want to train MaskRCNN you can have a look at this repo: https://github.com/facebookresearch/maskrcnn-benchmark

Fabien
